src/app/registration/registration.style.scss

```diff
@@ -0,0 +1,19 @@
+form {
```

Comment:
!fix use BEM-like methodology for styling https://github.com/garage-it/SmartHouse-frontend/wiki/HTML-style-guide not cascading